Hudl.Mjolnir/Command/CommandContext.cs

```diff
@@ -84,6 +96,24 @@ public static IStats Stats
             }
         }
 
+        public static void IgnoreExceptions(HashSet<Type> types)
```

Comment:
One of our internal libraries does a bit of Mjolnir initialization, wiring it into our service locator. I'm thinking it'll go there.